ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -72,10 +72,14 @@ function gather_bootstrap_and_fail() {
     GATHER_BOOTSTRAP_ARGS+=('--bootstrap' "${bootstrap_0_ip}")
     GATHER_BOOTSTRAP_ARGS+=('--master' "${control_plane_0_ip}" '--master' "${control_plane_1_ip}" '--master' "${control_plane_2_ip}")
 
-    set -e
```

Comment:
Removing this is going to leak out the `set +e` change. We need to call `set -e` at some point before returning from the function.

Reply:
Removed the `set +e` in the function. It isn't needed.